allauth/socialaccount/providers/apple/provider.py

```diff
@@ -27,5 +40,10 @@ def extract_email_addresses(self, data):
             )
         return ret
 
+    def get_default_scope(self):
+        scopes = ["name"]
+        if QUERY_EMAIL:
+            scopes.append("email")
```

Comment:
@benhowes Is it necessary to add email in the scope? Does not we get the email automatically in the identity data encoded in `id_token`?

Reply:
In  it says:
> "If you request the user’s verified email address, Sign in with Apple prompts the user for it, to share with your app. The user may choose to share their real email address or an anonymous one that uses the private email relay service."

I think it's safe to assume that we should include the scope if `QUERY_EMAIL` is set?

Reply:
I've just confirmed that the email is not included in the `id_token` without this scope being present.

Reply:
@benhowes Thanks for the verification.

Reply:
So is that all? Can we merge changes to original PR? :)